app/assets/stylesheets/components/_discussion.css.sass

```diff
@@ -23,38 +26,129 @@ $discussion-item-triangle-size: 40px
   flex-grow: 1
   color: theme-palette(primary)
 
-.Discussion-commentsCount
-  @include delta
+.Discussion-commenters
+  display: none
+
+.Discussion-info
   position: absolute
+  top: 0
   bottom: 0
   right: 0
-  z-index: 55
-  height: 17px
-  padding-right: 5px
+
+  display: flex
+  flex-direction: column
+  align-items: flex-end
+
+.Discussion-author
+  @include gamma
+
+.Discussion-author.Discussion-author--small
+  width: $discussion-inside-info-width
   color: theme-palette(neutral, white)
-  text-align: right
+  text-align: center
+  
+.Discussion-author.Discussion-author--big
+  display: none
 
-.Discussion-commentsCount:before
-  content: ''
-  position: absolute
-  right: 0
-  bottom: 0
-  z-index: -1
-  width: 0
-  height: 0
-  border-style: solid
-  border-width: 0 0 $discussion-item-triangle-size $discussion-item-triangle-size
-  border-color: transparent
+.Discussion-commentsCount
+  @include delta
+  width: $discussion-inside-info-width
+  margin-top: 1px
+  color: theme-palette(neutral, white)
+  text-align: center
 
 .Discussion.Discussion--watching,
 .Discussion.Discussion--watched
-  .Discussion-commentsCount:before
-    border-bottom-color: theme-palette(primary)
+  .Discussion-commentsCount
+    background-color: theme-palette(primary)
+  .Discussion-afterCommentsCount
+    border-color: theme-palette(primary)
 
 .Discussion.Discussion--unwatched
-  .Discussion-commentsCount:before
-    border-bottom-color: theme-palette(accent-secondary)
+  .Discussion-commentsCount
+    background-color: theme-palette(accent-secondary)
+  .Discussion-afterCommentsCount
+    border-color: theme-palette(accent-secondary)
 
 .Discussion.Discussion--unread
-  .Discussion-commentsCount:before
-    border-bottom-color: theme-palette(accent-primary)
+  .Discussion-commentsCount
+    background-color: theme-palette(accent-primary)
+  .Discussion-afterCommentsCount
+    border-color: theme-palette(accent-primary)
+
+@media only screen and (min-width : 992px)
+  .Discussion
+    padding-right: 0.5rem
```

Comment:
Good point. It was defined in the theme file as `padding: $default-spacing / 2`. I'll change this to match that.

Since I'm re-overriding this property to make it the default again, would it be better to do it the other way around, by only overriding `padding-right` in small screens?